thinc/backends/_custom_kernels.py

```diff
@@ -459,7 +467,7 @@ def backprop_gelu(
     return out
 
 
-def backprop_maxout(dY, which, P, threads_per_block=128, num_blocks=128):
+def backprop_maxout(dY, which, P, *, threads_per_block=128, num_blocks=128):
```

Comment:
If we are dealing with the kwargs already and we are creating potential backward compatibility issues, what about the variable name `P`? To me its not really descriptive. But maybe its descriptive enough because the number of pieces in the maxout layer is `nP` ?

Reply:
> creating potential backward compatibility issues

This PR should not create backwards compat issues, since `_custom_kernels` is a private module. If we'd change the naming in the public ops interface, we'd break backwards compat, not sure if it's worth doing that.

Reply:
Yea no I think it's fine as is